ZeroWasteFriendly/Scenes/Library/LibraryController.swift

```diff
@@ -82,3 +101,23 @@ extension LibraryController: UITableViewDelegate {
         postsDataSource[indexPath.row].expanded ? UITableView.automaticDimension : CGFloat(Constants.Library.defaultRowHeight)
     }
 }
+
+extension LibraryController: UIScrollViewDelegate {
+
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        let topPosition = tableView.contentOffset.y <= 0
+        interactor.didScrollTableView(topPosition: topPosition)
```

Comment:
interactor should not have methods named like delegates
you'd better call it handleTableViewScroll

- wrong: naming based on event that invoked it
- ok: naming on methods it does